pkg/kubelet/cm/devicemanager/pod_devices.go

```diff
@@ -346,3 +346,21 @@ func (pdev *podDevices) getContainerDevices(podUID, contName string) []*podresou
 	}
 	return cDev
 }
+
+// ResourceDeviceInstances is a map ping resource name -> device name -> device data
+type ResourceDeviceInstances map[string]map[string]pluginapi.Device
```

Comment:
I think we can come up with a better name for this (though I don't have any suggestions off the top of my head). Possibly something with the name `plugin` in it instead of `resource` to make it clear that this holds `pluginapi.Devices`. Maybe even just `PluginDevices`. Or maybe just `DeviceSet` to mirror `CPUSet`. Not sure.

Let's do that in a follow-up PR though.

Reply:
Yes, not fond of the name either. `DeviceSet` is tempting, but I think the name should convey there are nested maps. Will think about this.